apps/admin-gui/src/app/vos/components/application-list-details/application-list-details.component.ts

```diff
@@ -54,7 +55,7 @@ export class ApplicationListDetailsComponent implements OnChanges {
 
   addedColumns = new Set<string>();
 
-  @ViewChild(MatPaginator) paginator: MatPaginator;
+  @ViewChild(TableWrapperComponent, {static: true}) child: TableWrapperComponent;
```

Comment:
with your PR in this table some weird console errors are appearing
![image](https://user-images.githubusercontent.com/32871167/122234374-1ddb8a80-cebd-11eb-8565-c8b40dda151a.png)